lib/ddtrace/contrib/active_job/events/enqueue_at.rb

```diff
@@ -27,6 +27,7 @@ def span_name
           def process(span, event, _id, payload)
```

Comment:
Clearly not for this PR but WOW do we repeat a lot of stuff for every event for every thing in Rails. I think there's an opportunity here to perhaps create a class that can be parameterized and have all the common bits centralized. If we had such a thing, possibly hundreds of changes on this PR would probably be reduced to a single line change on this common class.